pitch.md

```diff
@@ -23,38 +35,187 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
+a hand diagram in text format might make sense for the output. Something like
+```
+        xx
+        xx
+        xxxxx
+        xxxx
+xxxx           xxx
+xxxx           xxxx
+xxxx           x
+x              xxxxx
+
+        xxxx
+        xxx
+        xxx
+        xxx
+```
+where x's are replaced with the actual cards.
 
 ## Operation (semantics)
 _What might happen when a program runs? Is there a computational model that
 corresponds with your domain? How does a program interact with the
 user? What kinds of semantic (i.e., non-syntax) errors might occur, and how 
 might they be communicated to the user?_
 
+When a program runs it will take the input by the user and display it in a nice
+manner.  This might consist of creating a bridge diagram based off the input. It
+could also be showing some end position -- that is showing the state of the cards
+after some number of cards have been played. Another useful thing a user could do
+when running the program would be to show some of the hands but not others. This
+would be useful for sharing a hand that they want to ask someone how they would
+play it without giving away what the right answer is (you could determine the
+right answer (for some definition of right) if you could see the opponents hand).
+Another use would be to show some or all of the hands a part of the bidding sequence
+(remember that bidding is what players do during the "auction" phase of the game)
+in order to query other bridge players about what they would have bid without
+revealing what they actually did. I think it would be very cool if you could write
+in a hand in a way that is convenient for you and then run the program to show the
+hand or whatever part of the hand you want to show in a way that is convenient for
+others. I would like this format to be easily sharable. One place I wish I could
+share bridge hands easily is on social media like Facebook, where currently I think
+the best way to share a hand is by sharing an image of a hand, which seems a bit
+clunky to me.
+
+After everything above is done to satisfaction a nice operation to be able to do
+would be to be able to do a "double dummy" play analysis.  That is to say, determine
+the optimal result if people could play with perfect information (see the other
+players cards).  I think this is not critical to the language, since a purely
+descriptive language would be very useful on its own.  However, this would make
+the language much more powerful and still keep it in the domain of bridge hand
+review.  It would also be desirable to do this analysis at any point during the
+hand. That is to say, to do this analysis after a certain number of tricks have
+been played in a non optimal manner.
+
+I believe this is data description computational model since it is mainly displaying
+the information the user gave it or pieces of information the user gave it in
+nice ways.
+
+A common syntax error that you will find in bridge books is players not having
```

Comment:
Would you have any checks on the users not replaying cards? Would it be okay for me to say that a hand is the cards: Ace of Spaces, 7 of Diamonds, Two of Hearts, 7 of Diamonds? Or would it be okay for someone to have played two of the same cards in subsequent turns? What would the error message say to help the user?